### PR TITLE
Apply CodeFormatter rules to branch

### PR DESCRIPTION
**Pull request by michaelcfanning**

### CODE REVIEW

**Comment by michaelcfanning** [comment]
@lgolding can you take a look at this?

File: src/Microsoft.DotNet.DeadRegionAnalysis/IAnalysisLogger.cs [hunk 1]
```diff
@@ -1,4 +1,7 @@
-﻿using System;
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```
Comment (AlexGhiondea) [inline src/Microsoft.DotNet.DeadRegionAnalysis/IAnalysisLogger.cs]: Why is the reference to System added?

Comment (lgolding) [inline src/Microsoft.DotNet.DeadRegionAnalysis/IAnalysisLogger.cs]: He didn't add the reference, he added the copyright notice above it.

**Comment by lgolding** [comment]
:+1:

**Comment by michaelcfanning** [comment]
@jaredpar @Priya91

**Comment by Priya91** [comment]
LGTM

File: src/Microsoft.DotNet.DeadRegionAnalysis/IAnalysisLogger.cs (hunk 1 above)
Comment (AlexGhiondea) [inline src/Microsoft.DotNet.DeadRegionAnalysis/IAnalysisLogger.cs]: Duh! :) Thanks!

**Pull request merged**